### PR TITLE
Add google_webmaster collector.

### PR DESCRIPTION
**Pull request by alienth**

### CODE REVIEW

File: cmd/scollector/collectors/google_analytics.go [hunk 1]
```diff
@@ -183,16 +183,17 @@ func getPageviews(md *opentsdb.MultiDataPoint, svc *analytics.Service, site conf
 	return nil
 }
 
-// analyticsClient() takes in a clientid, secret, and a base64'd gob representing the cached oauth token.
-// Generating the token is left as an exercise to the reader. (TODO)
-func analyticsClient(clientid string, secret string, tokenstr string) (*http.Client, error) {
+// googleAPIClient() takes in a clientid, secret, a base64'd gob representing
+// the cached oauth token, and a list of oauth scopes.  Generating the token is
+// left as an exercise to the reader.
```
Comment (kylebrandt) [inline cmd/scollector/collectors/google_analytics.go]: Why?

**Comment by kylebrandt** [comment]
sorry to be more clear, I meant why not document the process?

**Comment by alienth** [comment]
@kylebrandt because I don't remember how to do it. I'm just reusing the token I generated last time.

Could look into how that is done and document how to do it, but that's out of scope for this project.

**Comment by kylebrandt** [comment]
Fair enough - code looks good to me

**Pull request merged**